appc.js

```diff
@@ -9,11 +9,13 @@ module.exports = {
     platforms: {
         ios: {
             vendor: {
+
+                // Q: How would we make sure this is unique accross modules?
                 'MyFramework': {
                      // these can be an array or string
-                     source: ['src'],
-                     header: 'src',
-                     resource: 'src'
+                     source: ['vendor'],
+                     header: 'vendor',
+                     resource: 'vendor'
```

Comment:
You are right, but we still should follow the same naming conventions for both projects and modules. In addition, we should (maybe in 6.0.0) remove the tiapp.xml completely and move to the appc.js for that purpose. 

We discussed the problem of same frameworks with different versions in the last 6.0.0 meeting, @cbarber would be able to answer that question for you :-)